core/src/test/java/org/apache/druid/java/util/common/GranularityTest.java

```diff
@@ -809,6 +811,20 @@ public void testIncrementOverSpringForward()
     );
   }
 
+  @Test
+  public void testIsFinerComparator()
+  {
+    Assert.assertTrue(Granularity.IS_FINER_THAN.compare(NONE, SECOND) < 0);
+    Assert.assertTrue(Granularity.IS_FINER_THAN.compare(SECOND, NONE) > 0);
+    Assert.assertTrue(Granularity.IS_FINER_THAN.compare(NONE, MINUTE) < 0);
+    Assert.assertTrue(Granularity.IS_FINER_THAN.compare(MINUTE, NONE) > 0);
+    Assert.assertTrue(Granularity.IS_FINER_THAN.compare(DAY, MONTH) < 0);
+    Granularity day = DAY;
```

Comment:
nit: why is this a variable?

Reply:
Can you add 
    Assert.assertTrue(Granularity.IS_FINER_THAN.compare(NONE, NONE)
    Assert.assertTrue(Granularity.IS_FINER_THAN.compare(ALL, ALL)
    too?

Reply:
I had to create a variable because the comparator complained that it was being used against itself...

Reply:
Added a comment to explain why a variable is needed

Reply:
Added